<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HChromatismEditWidget</class>
 <widget class="QWidget" name="HChromatismEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="9">
    <widget class="HEntireTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_1">
     <property name="text">
      <string>添 加</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>删 除</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>计算g</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>计算ab</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HEntireTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">HePlugin/HEntireTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
